site/src/pages/CreateWorkspacePage/CreateWorkspacePage.test.tsx

```diff
@@ -185,6 +185,54 @@ describe("CreateWorkspacePage", () => {
     expect(validationError).toBeInTheDocument();
   });
 
+  it("external auth authenticates and succeeds", async () => {
+    jest
+      .spyOn(API, "getWorkspaceQuota")
+      .mockResolvedValueOnce(MockWorkspaceQuota);
+    jest
+      .spyOn(API, "getUsers")
+      .mockResolvedValueOnce({ users: [MockUser], count: 1 });
+    jest.spyOn(API, "createWorkspace").mockResolvedValueOnce(MockWorkspace);
+    jest
+      .spyOn(API, "getTemplateVersionExternalAuth")
+      .mockResolvedValue([MockTemplateVersionExternalAuthGithub]);
```

Comment:
Why not use msw? I thought you had suggested we use it in favor of jest spies.

Reply:
because this isn't actually a new test, it was from another branch. been cleaned up now. 😄